main/src/main/scala/sbt/Keys.scala

```diff
@@ -487,6 +488,8 @@ object Keys {
   val pluginData = taskKey[PluginData]("Information from the plugin build needed in the main build definition.").withRank(DTask)
   val globalPluginUpdate = taskKey[UpdateReport]("A hook to get the UpdateReport of the global plugin.").withRank(DTask)
   private[sbt] val taskCancelStrategy = settingKey[State => TaskCancellationStrategy]("Experimental task cancellation handler.").withRank(DTask)
+  private[sbt] val cacheStoreFactory = AttributeKey[CacheStoreFactoryFactory]("cache-store-factory")
+  val fileCacheSize = settingKey[Long]("The approximate maximum size in bytes of the cache used to store previous task results.")
```

Comment:
Should we should provide `parseBytes` function that can be used as `Defaults.parseBytes("128M")`  like

https://github.com/lightbend/config/blob/558c1e22f53c7f165daaf3e1e792399087d79f93/config/src/main/java/com/typesafe/config/impl/SimpleConfig.java#L851-L852

Reply:
I agree. I'm just trying to think of a better place to put it than Defaults.

Reply:
I ended up making the setting a String and leaving the parser private[sbt]: https://github.com/sbt/sbt/pull/4850/commits/80dc44a5051a53d104472f4a6c188dd7bd337e69